common/src/main/java/org/apache/rocketmq/schema/registry/common/storage/StorageServiceProxy.java

```diff
@@ -84,7 +86,7 @@ public void delete(final QualifiedName name) {
      * @param schemaInfo schema information instance
      * @return true if errors after this should be ignored.
      */
-    @CacheEvict(key = "'schema.' + #name.getTenant() + '/' + #name.schema()")
```

Comment:
Does here also follow the design of tenant/schema?

Reply:
Get it.